### PR TITLE
Issues #1 nad #7

### PR DESCRIPTION
**Pull request by ladyvargas**

### CODE REVIEW

File: content/settings.json [hunk 1]
```diff
@@ -2,10 +2,13 @@
   "title": "DMSE Documentation",
   "url": "",
   "logo": {
-    "light": "./icon1.png",
-    "dark": "./icon1.png"
+    "light": "./oma2.png",
```
Comment (jpradocueva) [inline content/settings.json:5]: Do we need these keys of "light" and "dark"? Why are they needed?

File: content/en/index.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ features:
   - Feature 2
   - Feature 3
 footer: MIT Licensed | Copyright © 2018-present Evan You
```
Comment (jpradocueva) [inline content/en/index.md:10]: We should change the footer license to:
footer: OMA Licensed | Copyright (c) 2021

**Review by jpradocueva — COMMENTED** [review]
See comments embeded inline.

File: content/settings.json (hunk 1 above)
Comment (jpradocueva) [reply content/settings.json:5]: Both logos, dark and light are used when changing the screen display from dark to light presentation mode.

**Review by jpradocueva — APPROVED** [review]

**Review by jpradocueva — APPROVED** [review]
@ladyvargas  ok to merge

**Review by ladyvargas — COMMENTED** [review]
@jpradocueva #8

**Comment by jpradocueva** [comment]
@lady it is ok to merge

**Pull request merged**